packages/tailwindcss/src/variants.ts

```diff
@@ -904,3 +908,16 @@ export function createVariants(theme: Theme): Variants {
 
   return variants
 }
+
+function normalizeAttributeSelectors(value: string) {
```

Comment:
I like `quoteAttributeValue` or `ensureQuotedAttributeValue`. I personally don't mind longer names but in this case the fact that it might not add quotes if there are quotes already are easy enough I think in that it won't be needed to communicate. 👍

Reply:
I think I like `quoteAttributeValue` the most

Reply:
Yeah me too! I changed the code to say that 👍  TY